main/src/test/java/org/mobilitydata/gtfsvalidator/validator/StopTimeTravelSpeedValidatorTest.java

```diff
@@ -298,6 +301,8 @@ public void farStopsMissingTimeInBetween_yieldsNotice() {
                 GtfsTime.fromString("08:02:30")));
     assertThat(generateNotices(ImmutableList.of(route), ImmutableList.of(trip), stopTimes, stops))
         .containsExactly(
+            createFastTravelBetweenConsecutiveStopsNotice(
```

Comment:
Since we now skip stops when we can't compute the distance or speed, we sometimes effectively check non-consecutive stops, which feels slightly misleading. If needed I can adjust the logic to only emit a notice for strictly consecutive entries. 

cc: @davidgamez @emmambd